workloads/templates/workload-nodevertical-script-cm.yml.j2

```diff
@@ -61,7 +53,7 @@ data:
             - num: {{total_pod_count.stdout|int}}
               image: {{nodevertical_pod_image}}
               basename: pausepods
-              file: pod-pause-labeled-nodes.json
+              file: pod-pause-labeled-nodes.yaml
```

Comment:
This file will be in the same `/root/workload/` directory as the CL config?

Reply:
Yup it is placed in the same directory as the cluster loader config file.

Reply:
I changed it so all three workloads consistently refer to their associated files in the same manner (`file: file_name.yaml` instead of mixed with `file: ./file_name.yaml`)